### PR TITLE
Improve customize.yaml non-inclusion warning message

### PR DESCRIPTION
**Pull request by scop**
Link to relevant docs, note that changes won't be applied after a
restart if the include isn't in place (changes _do_ actually affect the
current runtime configuration and are persisted in customize.yaml).

### CODE REVIEW

File: src/panels/config/customize/ha-form-customize.js [hunk 1]
```diff
@@ -26,9 +26,10 @@ class HaFormCustomize extends PolymerElement {
         if="[[computeShowWarning(localConfig, globalConfig)]]"
       >
         <div class="warning">
-          It seems that your configuration.yaml doesn't properly include
-          customize.yaml<br />
-          Changes made here won't affect your configuration.
+          It seems that your configuration.yaml doesn't properly
+          <a href="https://www.home-assistant.io/docs/configuration/customizing-devices/#customization-using-the-ui" target="_blank">include customize.yaml</a>.<br />
+          Changes made here are written in it, but will not be applied after a
```
Comment (bramkragten) [inline src/panels/config/customize/ha-form-customize.js]: remove the `after a restart` it is not necessary to do a restart when making changes through the UI. (and there is a service for reloading customize `homeassistant.reload_core_config`)

Comment (scop) [reply src/panels/config/customize/ha-form-customize.js]: Removing that will send the wrong message. Pretty much the whole point of this change is that changes **are** applied immediately. But I'll change that to "after a configuration reload".

Comment (bramkragten) [reply src/panels/config/customize/ha-form-customize.js]: Sorry missed the part about current run time, that would actually be a bug I think...

Comment (scop) [reply src/panels/config/customize/ha-form-customize.js]: I think it's fine as is, just the warning message needs a tweak.

**Review by bramkragten — APPROVED** [review]
OK to merge when travis passes.

**Pull request merged**